code/game/machinery/scanner_gate.dm

```diff
@@ -73,6 +82,8 @@
 
 /obj/machinery/scanner_gate/examine(mob/user)
 	. = ..()
+
+	. += span_notice("It's set to scan for [span_boldnotice(scangate_mode)]")
```

Comment:
This information doesn't need to be revealed unless unlocked, imo

Reply:
It is already visible when unlocked, this is why I made it also visible when not unlocked.

I want antags and such to know what this scanner detects, and if they should fear it (Weapons, Contraband) or not (Infections, Nutrition)